### PR TITLE
feat(colors): change design token colors to be more similar to GitHub's UI

### PR DESCRIPTION
**Pull request by YossiSaadi**
[CU-1x0typ9](https://app.clickup.com/t/1x0typ9), per to Shuju's request

### CODE REVIEW

File: public/styles/dark.css [hunk 1]
```diff
@@ -23,7 +23,7 @@
     --color-label-tags: var(--TBD);
     --color-label-shortcut: var(--TBD); */
 
-    --text-color-primary: var(--harbor-gray);
+    --text-color-primary: var(--pearl-river);
```
Comment (yhamry) [inline public/styles/dark.css:26]: This is dangerous - please verify this change very good !

File: public/styles/dark.css [hunk 2]
```diff
@@ -47,7 +47,7 @@
     --color-decorative-4: var(--high-violet);
     --color-decorative-5: var(--deep-sea);
 
-    --color-doc-cover: var(--gravel-rocks);
+    --color-doc-cover: var(--sea-anchor);
```
Comment (yhamry) [inline public/styles/dark.css:50]: Same here - you must verify this change doesn't break any of its usages.

**Review by yhamry — CHANGES_REQUESTED** [review]
Verify changes - see comment

File: public/styles/dark.css (hunk 1 above)
Comment (YossiSaadi) [reply public/styles/dark.css:26]: Thanks for the comment and the warning! 👍🏼 
Checked it on many places now, a big lighter color. An almost invisible change if you are not told it's been changed

File: public/styles/dark.css (hunk 2 above)
Comment (YossiSaadi) [reply public/styles/dark.css:50]: Checked also this one on most of the places, it seems cool, a bit darker color.

**Comment by YossiSaadi** [comment]
@yhamry Thanks for the heads up!
I checked and it seems cool :)

**Review by yhamry — APPROVED** [review]
Salamtak

**Pull request merged**